.github/workflows/publish-cdn.yml

```diff
@@ -0,0 +1,59 @@
+name: 'Publish: Publish to CDN'
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'The repository version to publish'
+        required: true
+        type: string
+
+  # This workflow is intended to be called from the primary publish workflow
+  # automatically. The manual trigger is maintained for partially failed workflows
+  # or edge cases that require this be run independently.
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The repository version to publish'
+        required: true
+        type: string
+
+env:
+  GH_ACTION_ROLE: arn:aws:iam::073865013699:role/CDN_Autocomplete_UI_GHA_Role
+  AWS_REGION : us-east-1
+
+jobs:
+  publish-to-cdn:
+    name: Publish to CDN
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.x
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build compiled package for publishing
+        run: npm run compile
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
**Suggestion**: We should use a fixed commit sha here.

Reply:
Done! 8db92ca7dca340405e2a7d5ec6c44faa0d424e99